lib/template-loader.php

```diff
@@ -199,7 +199,7 @@ function gutenberg_find_template( $template_file ) {
 			}
 		}
 		$_wp_current_template_id      = $current_template_post->ID;
-		$_wp_current_template_content = $current_template_post->post_content;
+		$_wp_current_template_content = empty( $current_template_post->post_content ) ? 'Empty template.' : $current_template_post->post_content;
```

Comment:
@epiqueras This string should likely be wrapped in a translation function like `__(`.

Reply:
#20277